### DISCUSSION TITLE
Highlighting of wrapped lines

### DISCUSSION THREAD

**Discussion by levouh** [nvim-orgmode/orgmode#117] [Q&A] [answered]
Curious what the intended functionality of highlighting for wrapped "header-level" lines is, for instance:

![image](https://user-images.githubusercontent.com/31262046/139186812-4adc1b48-ef64-4ebc-b9a1-3fa5c00e0ed8.png)

when I manually "wrap" the line, the highlighting changes. Is this just me writing things that are too long for header lines and treating them as perhaps bullet points should be used?

An alternative is to use Vim's native wrapping of lines and something like `:h breakindent` to have the wrapped content match the indent of original line, however this only matches leading white space and _Org_ won't have leading white space as headers are like:

```
*** ...
```

Normal wrapping (without `:h breakindent`) is alright too, but gets a bit out of hand and makes it hard to tell which lines are which.

---

I imagine the answer is "just use proper bullets, treat headers in _Org_ files like `#` headings in _Markdown_ files, but was just curious.

**Comment by kristijanhusak** [comment, marked as answer]
That's just how Orgmode works. First line is like a title of the task, and details go inside.
